Lib9c/CurrencyExtensions.cs

```diff
@@ -13,10 +13,25 @@ public static Bencodex.Types.Dictionary Serialize(this Currency currency)
             IValue minters = currency.Minters is IImmutableSet<Address> m
                 ? new Bencodex.Types.List(m.Select<Address, IValue>(a => new Binary(a.ByteArray)))
                 : (IValue)Null.Value;
-            return Bencodex.Types.Dictionary.Empty
+            var serialized = Bencodex.Types.Dictionary.Empty
                 .Add("ticker", currency.Ticker)
                 .Add("minters", minters)
                 .Add("decimalPlaces", new[] { currency.DecimalPlaces });
+            if (currency.TotalSupplyTrackable)
+            {
+                serialized = serialized.Add("totalSupplyTrackable", true);
+                if (currency.MaximumSupply.HasValue)
+                {
+                    serialized = serialized.Add(
+                        "maximumSupplyMajor",
+                        (IValue)new Integer(currency.MaximumSupply!.Value.MajorUnit)
+                        ).Add(
+                        "maximumSupplyMinor",
+                        (IValue)new Integer(currency.MaximumSupply!.Value.MinorUnit));
```

Comment:
Instead of using `!` keyword (null-check ignore), how about using `is` keyword? like:

```suggestion
                if (currency.MaximumSupply is { } maximumSupply)
                {
                    serialized = serialized.Add(
                        "maximumSupplyMajor",
                        (IValue)new Integer(maximumSupply.MajorUnit)
                        ).Add(
                        "maximumSupplyMinor",
                        (IValue)new Integer(maximumSupply.MinorUnit));
```